style.css

```diff
@@ -376,3 +390,177 @@ header a {
   color: #5e6c84;
   font-size: 17px;
 }
+
+.Desktop-version {
+  display: none;
+}
+
+/* Start of the desktop version */
+
+@media only screen and (min-width: 768px) {
+  .img-nav {
+    display: none;
+  }
+
+  .nav-bar ul {
+    display: flex;
+    gap: 16px;
+    align-items: center;
+  }
+
+  .skill-sets li {
+    color: #344563;
+    line-height: 24px;
+    padding-bottom: 30px;
+  }
+
+  .language-set li {
+    background-color: #f7f7f9;
+    width: 122px;
+    height: 120px;
+    border-radius: 8px;
+  }
+
+  .nav-bar ul li {
+    list-style: none;
+  }
+
+  .nav-bar li a {
+    text-decoration: none;
+    color: #344563;
+    font-size: 15px;
+    font-weight: 500;
+    font-family: "Poppins", sans-serif;
+    padding-left: 12px;
+  }
+
+  header {
+    padding: 25px 80px;
+  }
+
+  header p {
+    color: #6070ff;
+  }
+
+  .section-one {
+    background-image: url('images/header-image.png');
+    background-position: top left;
+    background-size: 99% 100%;
+    background-color: #fff;
+    padding: 80px 140px;
+    font-family: 'Poppins', sans-serif;
+  }
+
+  .section-one p {
+    font-size: 20px;
+    line-height: 28px;
+    max-width: 920px;
+  }
+
+  .section-one h1 {
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    color: #091e42;
+    letter-spacing: 0.37px;
+  }
+
+  .link-text {
+    color: #7f8cff;
+  }
+
+  .About-section {
+    display: flex;
+    align-items: center;
+    gap: 24px;
+    padding: 124px;
+    border-radius: 0;
+    flex: 1;
+  }
+
+  .about-wrap {
+    flex-basis: 30%;
+    align-items: flex-start;
+  }
+
+  .skill-sets {
+    flex-basis: 60%;
+  }
+
+  .arrows {
+    margin-bottom: 30px;
+    padding-top: 20px;
+  }
+
+  .language-set {
+    display: flex;
+    align-items: center;
+    gap: 12px;
+  }
+
+  .language-set li img {
+    padding-bottom: 25px;
+    margin-top: -15px;
+  }
+
+  .language-set span {
+    margin-top: 30px;
+    text-align: center;
+    margin-left: -25px;
+  }
+
+  .language-set .two {
+    order: 1;
+  }
+
+  .contact-section {
+    background-image: url(/images/contact\ form\ background\ shapesdesktop.png);
+    background-size: 90% 97%;
+    background-position: 80% 50%;
+    background-repeat: no-repeat;
+    padding: 80px 80px;
+  }
+
+  .contact-section p {
+    max-width: 569px;
+  }
+
+  .contact-input {
+    border-radius: 2px;
+    width: 447px;
+  }
+
+  .btn-contact {
+    margin: 0 auto;
+  }
+
+  .works-section {
+    padding: 70px 124px;
+  }
+
+  .work-card {
+    flex-direction: row;
+    gap: 24px;
+    border: 1px solid #a7aeff;
+    border-radius: 24px;
+    flex: 1;
+  }
+
+  .works-section .card-reverse {
+    flex-direction: row-reverse;
+    border: 2px solid #a7aeff;
+  }
+
+  .card-img {
+    flex-basis: 30%;
+  }
```

Comment:
Kindly ensure that the cards are designed as closely to the figma design as possible. The image and description both take up equal width.

Your design:
![image](https://user-images.githubusercontent.com/20554583/213388360-16fce1cd-897b-430e-b243-16339cf4f3e5.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/213388386-23573fed-e98e-4efe-be03-398aa56d5fef.png)